docs/src/main/asciidoc/quartz.adoc

```diff
@@ -400,6 +400,31 @@ After a few seconds, in another terminal, run `curl localhost:8080/tasks` to ver
 
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+[[quartz-register-plugin-listeners]]
+== Registering Plugin and Listeners
+
+You can register a plugin, jobListener or triggerListener through Quarkus configuration.
+
+The example bellow register the plugin `org.quartz.plugins.history.LoggingJobHistoryPlugin` named as `jobHistory` with the property `jobSuccessMessage` defined as `Job [{1}.{0}] execution complete and reports: {8}` 
```

Comment:
_The example bellow register**s** the plugin..._

Reply:
Thanks! I fixed it!